pixi.toml

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) ONNX Project Contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+[project]
+authors = ["ONNX Contributors <onnx-technical-discuss@lists.lfaidata.foundation>"]
+channels = ["conda-forge"]
+description = "Open Neural Network Exchange"
+name = "onnx"
+platforms = ["osx-arm64", "linux-64"]
+
+[tasks.install]
+cmd = 'export CMAKE_ARGS="$CMAKE_ARGS $CMAKE_EXTRA_ARGS" && pip install --no-deps --ignore-installed --verbose --no-build-isolation .'
+[tasks.install.env]
+ONNX_ML="1"
+ONNX_BUILD_TESTS="1"
+CMAKE_EXTRA_ARGS="-DONNX_USE_PROTOBUF_SHARED_LIBS=ON"
+
+[tasks.gen-docs]
+depends-on = "build"
+cmd = "python onnx/defs/gen_doc.py"
+
+[tasks.gtest]
+cmd = ".setuptools-cmake-build/onnx_gtests"
+
+[tasks.pytest]
+cmd = "pytest"
+
+[dependencies]
+c-compiler = ">=1.8.0"
+cmake = ">=3.31.5"
+cxx-compiler = ">=1.8.0"
+google-re2 = ">=1.1.20240702,<2"
+libdate = ">=3.0.1,<4"
+libprotobuf = "*"
+make = "*"
+ml_dtypes = ">=0.5.1"
+ninja = "*"
+numpy = ">=1.22.0"
+packaging = ">=24.1"
+parameterized = ">=0.9.0"
+pillow = ">=11.2.1,<12"
+pip = ">=25.0"
```

Comment:
the dependency list could probably improved: twine is not needed any more? in our other pipeline we use trusted publishing